### ISSUE TITLE
KDL-55EX725

### ISSUE THREAD

**Issue by LordMyschkin** [breunigs/bravia-auth-and-remote#6]
I am really sorry that there seems to be no way to make this work for my 2012 (?) KDL-55EX725 because of 404-Errors. Seems there  is a completly differnt site structure - are there any clues on how t ocontrol those TVs?

**Comment by breunigs** [comment]
I'm not aware of any good posts on that model, sorry. However, my model sent the commands without any encryption, so you should be able to capture the handshake and the commands sent when you set up the TV to work with the smartphone app. For Android, I used this app to capture the traffic: https://play.google.com/store/apps/details?id=jp.co.taosoftware.android.packetcapture&hl=en The pcap files it produces can be read with Wireshark.

**Comment by LordMyschkin** [comment]
I will try and - if I ever find a way - let you know, but I strongly doubt I am able to unterstand whatever wireshark will reveal me...

**Comment by breunigs** [comment]
If you shut off all other devices in your home network you could also share the pcap file.

**Comment by LordMyschkin** [comment]
Sorry, it took me quite some time to shut down everything connected to wlan in my household - i really hope I did not forget a device. 
here are four .pcaps, the last one should be to "cleanest" to analyze - I did see some packets in wireshark, but cannot translate them into some meaningful for me.  Since my tv is now my very last connected device not to be controled from my raspberry pi homemade home control page, I am now ready to get into this as deep as I can.

[Documents.zip](https://github.com/breunigs/bravia-auth-and-remote/files/464663/Documents.zip)

**Comment by breunigs** [comment]
It's been a while, but it's only now that I managed to get a look at it. From a quick glance there is nothing that jumps into my eye. Do you by any chance remember which IP address your TV had? (or, at least what is your local address space? I can see `10.8.x.x` and also `192.168.2.x`)

**Comment by LordMyschkin** [comment]
My adress range is 192.168...., and most likely, the tv has 192.168.2.202.
Thanks a lot for looking at it!

**Comment by breunigs** [comment]
Hm, I don't see anything familiar in those logs, or anything to looks like traffic to the TV. The only thing that seems vaguely promising is a device `10.8.0.1` that POSTs to `192.168.0.2` with `/sony/system HTTP/1.1  (application/json)`. 

How did you capture the traffic, i.e. where did wireshark run and from where did you issue the "remote control setup"? If wireshark was running on your computer and you tried to pair your phone with your TV, chances are Wireshark didn't receive any of the interesting traffic in the first place.

**Comment by LordMyschkin** [comment]
No, I captured with tPacketCapture running on the device i tried to pair; I will retry the experiment as soon as possible with a new device and without gms enabled - I guess the phone net is the cause of this 10.0 - Adress space...
Strange, it IS supposed to work somehow - since there is a software wich emulates a remote on windows via WLAN...

**Comment by hExPY** [comment]
Does anyone managed to fixed for kdl-XX725? Got the same error here.
